Scrapers/VenueClass.py

```diff
@@ -2,7 +2,7 @@
 class Venue:
     venue = {}
```

Comment:
TODO: Once all scrapers has been finalised, this file will be shifted to DataFormatting folder.